tf_files/aws/modules/utility-vm/cloud.tf

```diff
@@ -0,0 +1,198 @@
+resource "aws_cloudwatch_log_group" "csoc_log_group" {
+  name              = "${var.vm_hostname}"
+  retention_in_days = 1827
+
+  tags {
+    Environment  = "${var.environment}"
+    Organization = "Basic Services"
+  }
+}
+
+data "aws_ami" "public_cdis_ami" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu16-docker-base-1.0.2-*"]
+  }
+
+  owners = ["${var.ami_account_id}"]
+}
+
+resource "aws_ami_copy" "cdis_ami" {
+  name              = "ub16-cdis-crypt-1.0.2-nginx"
```

Comment:
Where does this AMI come from?
Could we just clone a public ubuntu16 image, and `apt install nginx`?
   https://cloud-images.ubuntu.com/locator/ec2/
Or (even better) run a docker? :-)
    https://hub.docker.com/_/nginx/
    https://docs.docker.com/install/linux/docker-ce/ubuntu/#set-up-the-repository

Reply:
This is the same AMI that is used for everything else, it comes already loaded with awslogs and it is easier to hook up to cloudwatch. We could copy a public Ubuntu AMI I suppose, I'll check out the outcome os said change today/.

Moreover, if we want to dockerize the service or not, it's got nothing to do with the image itself. Services and packages would be handled by a script in the flavors folder.